test/other/embind_jsgen_static_init.cpp

```diff
@@ -0,0 +1,23 @@
+#include <stdio.h>
+#include <fstream>
+#include <emscripten.h>
+#include <emscripten/bind.h>
+
+using namespace emscripten;
+
+class Foo {
+public:
+  void go(const std::string& filename) {
+    std::ifstream ifs(filename, std::ifstream::binary);
+    ifs.close();
+  }
+};
+
+int main() {
+  printf("done\n");
+}
+
+EMSCRIPTEN_BINDINGS(xxx) {
+    class_<Foo>("Foo")
+      .function("go", &Foo::go);
+}
```

Comment:
I don't see any static initializers in this file?  Should this test be call `method_pointers` or something like that?